gnmi/gnmit/gnmit_test.go

```diff
@@ -149,7 +149,7 @@ func toUpd(r *gpb.SubscribeResponse) []*upd {
 // TestONCE tests the subscribe mode of gnmit.
 func TestONCE(t *testing.T) {
 	ctx := context.Background()
-	c, addr, err := New(ctx, nil, "localhost:0", "local", false, nil)
+	c, addr, err := New(ctx, nil, "localhost:0", "local", false)
```

Comment:
these tests should stop the server at end, i think that cause the test to fail

Reply:
This hit some rare race condition, essentially we can't have listeners listening to it before we try to delete the socket file. I'll take a look at fixing this in a follow-up.

Reply:
sgtm